pkgs/os-specific/linux/minimal-bootstrap/stage0-posix/bootstrap-sources.nix

```diff
@@ -15,20 +18,63 @@ rec {
     executable = true;
```

Comment:
> While it is included in the stage0-posix source bundle and is synced with

Would you mind updating this comment while you're at it to reflect that this binary is no longer in the source bundle?

Reply:
908b2a51430e